File: GraphPlan (sgware)/src/edu/uno/ai/planning/gp/GraphPlan.java
package edu.uno.ai.planning.gp;

import edu.uno.ai.planning.Search;
import edu.uno.ai.planning.pg.PlanGraph;
import edu.uno.ai.planning.pg.PlanGraphPlanner;

public class GraphPlan extends PlanGraphPlanner {

	public GraphPlan() {
		super("SGP");
	}

	@Override
	protected Search makeSearch(PlanGraph graph) {
		System.out.println(graph.size());
		return new DepthFirstSearch(graph);
	}
}

File: UnitySokoban/Assets/Scripts/Planning/GraphPlanSGW/GraphPlan.cs
using PlanGraphSGW;
using Planning;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GraphPlanSGW
{
    public class GraphPlan : PlanGraphPlanner
    {
        public GraphPlan()
            :base("SGP")
        {
        }

        public override Search makeSearch(PlanGraph graph)
        {
            return new DepthFirstSearch(graph);
        }
    }
}
